packages/launchpad/src/components/code/FileRow.cy.tsx

```diff
@@ -120,4 +120,8 @@ describe('FileRow', () => {
 
     cy.percySnapshot()
   })
+
+  it('has expected styles', () => {
+    cy.get('h2').should('have.class', 'font-semibold')
+  })
```

Comment:
As a general rule, do we write style tests like this or do we just consider it "done" thanks to the percy snapshot?